pkg/cmd/server/api/types.go

```diff
@@ -998,3 +1001,26 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// ClusterResourceOverrideConfig is the configuration for the ClusterResourceOverride
+// admission controller which overrides user-provided container request/limit values.
+type ClusterResourceOverrideConfig struct {
```

Comment:
On Tue, Feb 9, 2016 at 8:04 AM, David Eads <notifications@github.com> wrote:
>
> This seems to be focused specifically on limits. I think I would have it
> match your variable name LimitRequestOverridesConfig.
>

@smarterclayton or anyone else want to chime in? I don't have a strong
opinion - hard to think of any name that won't be at least mildly
misleading to somebody. But I only want to change this one more time this
week :)